schema/defs.json

```diff
@@ -23,6 +23,14 @@
         "Env": {
             "$ref": "#/definitions/ArrayOfStrings"
         },
+        "mapStringString": {
```

Comment:
Question: For Kubernetes / Container annotations there are limits to the lenghts of the fields (or at least the name). Do we have any such restrictions that we should add?

Reply:
In the case of annotations, the key has a limit of 63 characters or less and all the other constraints that are valid for labels too, the values can be arbitrary text  and the sum of all keys + values are not allowed to exceed 256 kB (https://github.com/kubernetes/kubernetes/blob/98742f9d77a57aec44cc05b1daf721973fb029be/staging/src/k8s.io/apimachinery/pkg/api/validation/objectmeta.go#L36). I am not clear if we need a prefix for the per device or if we add per spec annotations that are not changing the Pod spec.

Reply:
@elezar Added the check for annotations PTAL

Reply:
Thanks, I'll give it a look. Also note that we're looking at determining the "minimum spec version" in #93 so that producers can fill the information that they need and then save the spec with the minimum version. We would need to add a function for checking whether a `v0.6.0` spec is required in the case of annotations. If this gets merged before then, I can add that though.